Sprint-1/interpret/time-format.js

```diff
@@ -12,13 +12,20 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
-
+//there are 6 varibale declarations
 // b) How many function calls are there?
-
+//There is one function call, console.log
 // c) Using documentation, explain what the expression movieLength % 60 represents
+// according to the documentation the The remainder (%) operator returns the remainder left over when one operand is divided by a second operand. It always takes the sign of the dividend.
 
+// The remaining value of the movieLength will be return as the remaining amounts of minutes since the saved 60 minutes will be equivalent to an hour
 // d) Interpret line 4, what does the expression assigned to totalMinutes mean?
 
+//The total minutes line takes all the seconds thats are able to fit into miutes and saves that and gives the reminder secconds that dont fit full into 60 seconds which counts as a whole minute.
```

Comment:
Try to feed the code to ChatGPT to see how else the code can be explained. (Explaining code can be challenging! It takes a lot of practice)

Reply:
Hi @cjyuan I made changes to the time-format.js. I tried using GPT to give me clearer explanations, I think they are a bit more straightforward than the ones I currently have